src/vs/platform/terminal/test/common/capabilities/commandDetection/promptInputModel.test.ts

```diff
@@ -55,6 +55,7 @@ suite('PromptInputModel', () => {
 		const cursorIndex = valueWithCursor.indexOf('|');
 		strictEqual(promptInputModel.value, value);
 		strictEqual(promptInputModel.cursorIndex, cursorIndex, `value=${promptInputModel.value}`);
+		assert.ok(promptInputModel.ghostTextIndex === -1 || cursorIndex <= promptInputModel.ghostTextIndex);
```

Comment:
Ok gives pretty bad failure messages, so this would help make the failure more actionable:

```suggestion
		assert.ok(promptInputModel.ghostTextIndex === -1 || cursorIndex <= promptInputModel.ghostTextIndex, `cursorIndex (${cursorIndex}) must be before ghostTextIndex (${ghostTextIndex})`);
```